### PR TITLE
chore: [] add upload scripts to examples

### PR DESCRIPTION
**Pull request by loweisz**
## Purpose
Adds the upload script to examples where it was missing. 

fixes https://github.com/contentful/apps/issues/2351

### CODE REVIEW

**Comment by andipaetzold** [comment]
Please double check the package-lock.json diffs

File: examples/commercelayer/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "1.5.89",
   "private": true,
   "devDependencies": {
-    "@contentful/app-scripts": "0.8.11",
+    "@contentful/app-scripts": "^0.8.11",
```
Comment (andipaetzold) [inline examples/commercelayer/package.json:6]: Why?

**Review by andipaetzold — COMMENTED** [review]
Why are you unpinning the dev deps?

File: examples/commercelayer/package.json (hunk 1 above)
Comment (loweisz) [reply examples/commercelayer/package.json]: It's pinned again 👍

Comment (andipaetzold) [reply examples/commercelayer/package.json]: Still unpinned in the other apps 😉

Comment (loweisz) [reply examples/commercelayer/package.json]: Not anymore 👍

**Review by andipaetzold — APPROVED** [review]

**Pull request closed without merging**